src/components/screens/FormScreen.tsx

```diff
@@ -3,35 +3,49 @@ import styles from "./FormScreen.module.css";
 import "@fontsource/instrument-serif";
 import "@fontsource/oxygen-mono";
 import Dices from "../icons/Dices";
+import { PaniniFormData } from "../enumFaces/enumFaces";
 import FormModuleBase from "../modularForm/FormModuleBase";
 import FormModuleExtras from "../modularForm/FormModuleExtras";
 import FormModuleFinal from "../modularForm/FormModuleFinal";
 import classNames from "classnames";
+import { useForm, SubmitHandler, FormProvider } from "react-hook-form";
 
 function FormScreen() {
-  const [isBeingTurnedOff, setisBeingTurnedOff] = useState(false)
+  const formMethods = useForm<PaniniFormData>();
+  const { handleSubmit } = formMethods;
 
-  const handleOrder = () => {
-    setisBeingTurnedOff(true)
-  }
+  const onSubmit: SubmitHandler<PaniniFormData> = (data) => {
```

Comment:
**nitpick:** data is not typed

Reply:
I believe the type for this data was created in the next PR